src/cpp/src/gguf_utils/gguf_modeling.cpp

```diff
@@ -152,24 +152,43 @@ std::shared_ptr<ov::Model> create_language_model(
 
 } // namespace
 
-std::shared_ptr<ov::Model> create_from_gguf(const std::string& model_path) {
+void save_openvino_model(const std::shared_ptr<ov::Model>& model, const std::string& save_path, bool compress_to_fp16) {
+    try {
+        auto serialize_start_time = std::chrono::high_resolution_clock::now();
+        ov::save_model(model, save_path, compress_to_fp16);
+        auto serialize_finish_time = std::chrono::high_resolution_clock::now();
+        auto serialize_duration = std::chrono::duration_cast<std::chrono::milliseconds>(serialize_finish_time - serialize_start_time).count();
+        std::cout << "Save generated OpenVINO model to: " << save_path << " done. Time: " << serialize_duration << " ms\n";
+    }
+    catch (const ov::Exception& e) {
+        std::cerr << "[Warning] Exception during model serialization: " << e.what() << std::endl;
```

Comment:
Please add a test to `tests/python_tests/test_llm_pipeline.py` to ensure we don't break the feature accidentally later

Reply:
Added test case